invenio_records_lom/fixtures/demo.py

```diff
@@ -0,0 +1,363 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+import json
+
+from faker import Faker
+
+
+# ----- functions for LOM datatypes -----
+def langstringify(fake: Faker, string: str) -> dict:
+    """Turn `string` into a LangString-object, as specified by LOMv1.0-standard."""
+    return {
+        "language": create_fake_language(fake),
```

Comment:
does this mean, that the whole record could have as many languages as often as langstringify is used? Because every new call could have set a new language string, but the faker always uses English to create the string?

Reply:
Yes. Well, a fake *language-code* (with optional subcodes) actually. The naming is due to LOMv1.0:
![image](https://user-images.githubusercontent.com/86989418/128832929-a77224bf-1eb9-4c9b-8fd6-016929fa4d04.png)
![image](https://user-images.githubusercontent.com/86989418/128833432-e1132b90-4997-4d55-9312-f8314014c4f1.png)

This is a left-over from when I tried to use faker's multiple-locale mode.
Right now, faker always uses the "en-US"-locale to fake things.
However, even just this one locale has multiple associated language codes, e.g. "en", "en-US", "en-US-philadelphia", ...
I could provide a language code randomly chosen from a hard-coded list of "en-US"-associated language codes instead?